src/formElementPlugins/listBuilder/src/ApplicationView.tsx

```diff
@@ -64,6 +75,18 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   const [open, setOpen] = useState(false)
   const [inputError, setInputError] = useState(false)
 
+  const [currentResponseElementsState, setCurrentResponseElementsState] = useState<any>()
```

Comment:
would prefer some type, even if it's `{[key: string]: any} | undefined`

Reply:
Cool: 29ab720d
```
const [currentResponseElementsState, setCurrentResponseElementsState] =
    useState<{ [key: string]: ElementState }>()
```